src/UI/Components/TreeTable/Helpers/TreeTableHelper.ts

```diff
@@ -66,16 +66,18 @@ export abstract class BaseTreeTableHelper<A extends AttributeTree>
       isExpandedByParent,
       isChildExpanded,
       createOnToggle,
-      createCloseAll,
-      createOpenAll,
       this.extractValues
     );
 
     const nodes = this.attributeHelper.getMultiAttributeNodes(this.attributes);
 
-    return Object.entries(nodes)
-      .map(([key, node]) => treeRowCreator.create(key, node))
-      .sort((a, b) => a.id.localeCompare(b.id));
+    return {
+      rows: Object.entries(nodes)
+        .map(([key, node]) => treeRowCreator.create(key, node))
+        .sort((a, b) => a.id.localeCompare(b.id)),
```

Comment:
if by any hasard "a" doesn't have an id, your code will break. 
I know it's handy to use such one liners, but in this case, I'd recommend to block it with an if case and return statement.